app/models/setting.rb

```diff
@@ -67,6 +67,15 @@ class Setting < RailsSettings::Base
       secret: ENV['GITLAB_SECRET'],
     }, validates: { json: { format: :hash } }
 
+    field :github, type: :hash, display: true, restart_required: true, default: {
+      enabled: ActiveModel::Type::Boolean.new.cast(ENV['GITHUB_ENABLED'] || false),
+      site: ENV['GITHUB_SITE'] || 'https://github.com/login/oauth/authorize',
+      scope: ENV['GITHUB_SCOPE'] || 'user,repo,gist',
```

Comment:
```suggestion
      scope: ENV['GITHUB_SCOPE'] || 'user,read:org',
```

Do not over-request permissions, right?